packageurl.go

```diff
@@ -482,11 +482,14 @@ func parseQualifiers(rawQuery string) (Qualifiers, error) {
 			return nil, fmt.Errorf("error unescaping qualifier value %q", value)
 		}
 
-		q = append(q, Qualifier{
-			Key: strings.ToLower(key),
+		if len(value) > 0 {
 			// only the first character needs  to be lowercase. Note that pURL is always UTF8, so we
 			// don't need to care about unicode here.
-			Value: strings.ToLower(value[:1]) + value[1:],
+			value = strings.ToLower(value[:1]) + value[1:]
```

Comment:
I am not even sure the lowercase first character is needed. I can't see it specified in the spec.

Reply:
@tommyknows  Hey Ramon, do you know if we really need lowercase characters here?

Reply:
Mh, I thought this was just the behaviour I ported from before the refactor, but I actually can't seem to find that 👀 

i wonder why I've added it in that case... 

Also, it's not in the spec and I can't see any test cases requiring this. I'd say remove it then...

Reply:
I am in favour of removing the lowercase conversion. I updated the title of the pull request and added another commit to just remove the case conversion.